tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Utils/Settings.cs

```diff
@@ -10,11 +10,30 @@ public sealed class Settings
 
         public string OrganisationApiBaseUrl { get; }
 
+        public SmtpServer Smtp { get; }
+
         public Settings(IConfigurationRoot config)
         {
             AdminConnectionString = config.GetConnectionString("CatalogueUsersAdmin");
             ConnectionString = config.GetConnectionString("CatalogueUsers");
             OrganisationApiBaseUrl = config.GetValue<string>("OrganisationApiBaseUrl");
+            Smtp = new SmtpServer(config.GetSection("SmtpServer"));
```

Comment:
You don't need to do all this - you can use something like config.GetSection<SmtpServer>("SmtpServer")

Reply:
I'm sorry, I don't follow. Would you please elaborate on this one / send some documentation that explains what you mean?

Reply:
`Smtp = config.GetSection("SmtpServer").Get<SmtpServer>();`

The framework will auto-magically deserialize the settings for you (providing properties have a `public` set and their names match the JSON names).

You can see an example in the send-email branch.

Reply:
That sounds nice, I always like a bit of auto-magic :D I was trying to stay away from having the config have sets on them, but what you proposed sounds tempting. I'll reconsider.